ui-v2/app/services/repository/type/event-source.js

```diff
@@ -65,7 +65,7 @@ const createProxy = function(repo, find, settings, cache, serialize = JSON.strin
         key: key,
         type: BlockingEventSource,
         settings: {
-          enabled: settings.blocking,
+          enabled: typeof settings.blocking === 'undefined' || settings.blocking,
```

Comment:
If settings.block is undefined / not set, this would enable it, otherwise if it's set to something it'll respect that setting, which makes sense.

Reply:
👌